docs/src/main/asciidoc/doc-reference.adoc

```diff
@@ -149,29 +149,36 @@ Minimally, each document should define and id and a title, and include common at
 <3> Include common document attributes.
 <4> Specify the relevant <<Categories>> (comma separated).
 
-==== Other common document attributes:
+==== Other common document header attributes:
 
 `:extension-status: preview`:: Use this attribute to flag special types of content. Valid values: `experimental`, `preview`, `stable` (not usually used), and `deprecated`.
 `:summary: <text>`:: Use the summary to provide a concise (26 words or less) description of the document. 
-This attribute will be used in tiles or other descriptions on the website. If not present, the contents of the <<Abstracts (preamble),preamble>> will be used instead.
+The value of this attribute is used in tiles or other descriptions on the website and is not required in newer diataxis-styled docs, as outlined in <<abstracts-preamble>>. 
+If not present, the contents of the <<Abstracts (preamble),preamble>> is used instead.
 
-NOTE: Take care with whitespace when working with document-scoped attributes. The document header ends with the first blank line. 
+IMPORTANT: Take care with whitespace when working with document-scoped attributes. The document header ends with the first blank line. 
 
+[[abstracts-preamble]]
 === Abstracts (preamble)
 
-Add a short description that helps your audience to quickly find and understand the purpose and intent of the page.
+The first paragraph in the main body is treated as the abstract, also referred to as the preamble.
+Add a short description that helps your audience quickly find and understand the purpose and intent of the page.
+The first sentence of the abstract provides a summary and gets automatically added to the tile on the link:https://quarkus.io/guides/[Quarkus guides homepage] unless there is a `:summary: <text>` attribute in the doc header.
```

Comment:
@michelle-purcell: Really well-explained update. Just a small comment to line 166. For maximum clarity (and in case the reader doesn't also read the Doc Header section), would it help to repeat here (perhaps as a note) that the summary attribute might not be present as it is not required in newer diataxis-styled docs.

Reply:
@sheilamjones - Thank you for reviewing. Good point :+1: 

@ebullient - What do you think? 

We know that `:attributes:` in the doc header can optionally be used in Quarkus docs to generate the blurb on the tile (as per non-diataxis styled content). 
But as @sheilamjones + :arrow_up:  correctly points out, the new templates do not include it at all and it makes more sense that our how-to and doc-reference encourage contributors to single-source the blurb on the tile from the 1st line of the Abstract (preamble) in the adoc source file. 

Otherwise, we have a mix of both approaches in diataxis styled topics and/or doc bugs on the tiles like we have had in the last month.

Reply:
yes. I agree that it is confusing. What we could think about is something like this: 

> The first sentence of the abstract provides a summary and gets automatically added to the tile on the link:https://quarkus.io/guides/[Quarkus guides homepage]. 
> If the first sentence is too long or can not otherwise be sensibly simplified to fit on the website tile, a `:summary:` attribute can defined in the document attributes to serve that purpose.

Or something. Make it clear that the summary attribute is a compensation/fallback thing.

Reply:
@ebullient - Thanks for reviewing. Yes, that is much clearer. I pushed another commit to modify as per :arrow_up: with some tweaks:

Line  183 -184:
>`If the first sentence is too long or can not be simplified to fit on the website tile, you can define a ``:summary:`` attribute in the document header attributes to serve that purpose. `
>`For more information, see <<doc-header-optional>>.`